README.md

```diff
@@ -51,7 +51,8 @@ After installing udata event orchestration package, you can run end-to-end tests
 
 Make sure you have your docker-compose with all services running and run:
 ```
-pytest
+pip install pytest
+pytest tests/
```

Comment:
I saw that I add added a `pytest.ini` with the testpaths but hadn't push.
But `pytest tests` may be more explicit and allow to run *all* tests instead also.